doc/modules/model_evaluation.rst

```diff
@@ -136,15 +152,12 @@ measuring a prediction error given ground truth and prediction:
 - functions ending with ``_score`` return a value to
   maximize, the higher the better.
 
-- functions ending with ``_error`` or ``_loss`` return a
+- functions ending with ``_error``, ``_loss``, or ``_deviance`` return a
   value to minimize, the lower the better.  When converting
   into a scorer object using :func:`make_scorer`, set
   the ``greater_is_better`` parameter to ``False`` (``True`` by default; see the
   parameter description below).
 
-Metrics available for various machine learning tasks are detailed in sections
-below.
-
 Many metrics are not given names to be used as ``scoring`` values,
 sometimes because they require additional parameters, such as
 :func:`fbeta_score`. In such cases, you need to generate an appropriate
```

Comment:
To remove those unwanted changes from this branch, you can try:

```
git checkout main -- doc/modules/model_evaluation.rst
```

and commit and push the results.

Reply:
@marijavlajic have you tried the above command? Doesn't it work as expected? It would be good to disantangle the contents from this PR from #28750.